packages/encryption/tests/messageSignature.test.ts

```diff
@@ -0,0 +1,19 @@
+import { decodeMessage, encodeMessage, hashMessage } from '../src/messageSignature';
+
+test('encodeMessage / decodeMessage', () => {
+  const message1 = 'Hello world';
+  const encodedMessage1 = encodeMessage(message1);
+  const decodedMessage1 = decodeMessage(encodedMessage1);
+  expect(decodedMessage1.toString()).toEqual(message1);
+
+  const message2 = '';
+  const encodedMessage2 = encodeMessage(message2);
+  const decodedMessage2 = decodeMessage(encodedMessage2);
+  expect(decodedMessage2.toString()).toEqual(message2);
+});
+
+test('hash message', () => {
+  const message1 = 'This is a new message';
```

Comment:
same here, should we compare w/ a real hash?
e.g. `d6d95e464da6763ca98a91cb4087f0e1c7c7870dd94cd8d651fccafe2124bbc1` generated from the first google result for sha256 online 😅

Reply:
I used openssl instead of a google result 😄 . Let me know if this is fine.

Reply:
Even better 😉